pom.xml

```diff
@@ -548,6 +548,26 @@
             </build>
         </profile>
         <profile>
+            <id>jdk8</id>
+            <activation>
+                <jdk>[1.8,1.9)</jdk>
+            </activation>
+            <build>
+                <pluginManagement>
+                    <plugins>
+                        <plugin>
+                            <groupId>org.apache.maven.plugins</groupId>
+                            <artifactId>maven-javadoc-plugin</artifactId>
+                            <configuration>
+                                <javadocVersion>1.8</javadocVersion>
+                                <additionalparam>-Xdoclint:all,-Xdoclint:-missing</additionalparam>
```

Comment:
Thanks.  Makes sense.


Get Outlook for Android






On Wed, May 11, 2016 at 6:43 AM -0700, "Christopher Tubbs" <notifications@github.com> wrote:





> @@ -548,6 +548,26 @@
>              </build>
>          </profile>
>          <profile>
> +            <id>jdk8</id>
> +            <activation>
> +                <jdk>[1.8,1.9)</jdk>
> +            </activation>
> +            <build>
> +                <pluginManagement>
> +                    <plugins>
> +                        <plugin>
> +                            <groupId>org.apache.maven.plugins</groupId>
> +                            <artifactId>maven-javadoc-plugin</artifactId>
> +                            <configuration>
> +                                <javadocVersion>1.8</javadocVersion>
> +                                <additionalparam>-Xdoclint:all,-Xdoclint:-missing</additionalparam>

The doclint check for **missing** is far too spammy with warnings. Almost no project develops with the principle that *everything* needs a full javadoc. So, I excluding checks for **missing**. If there are any that are missing that should be there, then we should add them, but we shouldn't be spammed with warnings about missing ones that aren't really needed.

If we really want to enable this check as well, I'm okay with it, but it should be done in conjunction with a changeset that adds all the ones that are currently missing.

---
You are receiving this because you modified the open/close state.
Reply to this email directly or view it on GitHub:
https://github.com/revelc/formatter-maven-plugin/pull/106/files/d88f08b51aa230ec5266f2197dd2bbc51f5d0b81#r62849530